recipes/dash_colorscales/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "dash_colorscales" %}
+{% set version = "0.0.4" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: e2eb8c78e702d03e9c13fb7ca888ed202152494dbb053f5aab6417b6f6d19f63
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python
+    - dash
+
+
+test:
+  imports:
+    - dash_colorscales
+
+about:
+  home: https://github.com/plotly/dash-colorscales
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
```

Comment:
Can you rename this to `LICENSE.txt` so it gets picked up from the source tarball when the `MANIFEST.in` file is fixed (https://github.com/plotly/dash-colorscales/pull/5).

Reply:
@chrisburr should be all set now, checks are just finishing up.  Good catch!